packages/clerk-js/src/ui/components/OrganizationSwitcher/UserMembershipList.tsx

```diff
@@ -21,12 +21,11 @@ export const UserMembershipList = (props: UserMembershipListProps) => {
 
   const { hidePersonal } = useOrganizationSwitcherContext();
   const { organization: currentOrg } = useCoreOrganization();
-  const { organizationList } = useCoreOrganizationList();
+  const { userMemberships } = useCoreOrganizationList();
   const user = useCoreUser();
 
-  const otherOrgs = (organizationList || []).map(e => e.organization).filter(o => o.id !== currentOrg?.id);
+  const otherOrgs = (userMemberships.data || []).map(e => e.organization).filter(o => o.id !== currentOrg?.id);
 
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
This will not work, tomorrow I will push a PR that will unblock this, for now can we leave it as is ?

Reply:
will be resolved with this: https://github.com/clerk/javascript/pull/2118

Reply:
I think this is now resolved, but @dimkl, can you please confirm?

Reply:
i think it's resolved. @panteliselef could you take a look and approve this PR (since it's blocked by "request changes") ?